envoy.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/google/shlex"
+)
+
+func runEnvoy(args string) error {
+	argParts, err := shlex.Split(args)
```

Comment:
What all are we planning on passing in practice? I imagined we'd have specific xds level flags that got passed through to envoy, not passing envoy specific flags.

Again, more along my vision of envoy being implementation behavior that's abstracted by xds. We accept config options in xds, and then it knows how to run envoy.